databaseFunctions.js

```diff
@@ -0,0 +1,72 @@
+const { db, end } = require("./database.js");
+
+// Takes in task name for creation and outputs the id of the created task
+async function addTask(taskName) {
+  try {
+    const query = `INSERT INTO task (name, completed, created_date)
+                        VALUES ($1, false, NOW())
+                        RETURNING id;`;
+
+    const res = await db.one(query, [taskName]);
+    return res;
+  } catch (err) {
+    console.error("Error adding in task: " + err);
+    throw err;
+  } finally {
+    end();
+  }
+}
+
+async function getTasks() {
+  try {
+    const query = `SELECT id, name, completed 
+                        FROM task;`;
+    const res = await db.manyOrNone(query);
```

Comment:
Since we're not doing anything with `res` before returning, could we directly return the result from `db.manyOrNone`? Do we even need async/await here? This applies to all of the queries here, not just this one.